src/audio/mixin_mixout/mixin_mixout.toml

```diff
@@ -37,6 +37,17 @@
 			7, 0, 0, 0, 296, 2700000, 48, 64, 0, 0, 0,
 			8, 0, 0, 0, 296, 2964000, 96, 128, 0, 0, 0,
 			9, 0, 0, 0, 296, 3492000, 192, 256, 0, 0, 0]
+#elif CONFIG_LUNARLAKE
```

Comment:
`#elif CONFIG_LUNARLAKE` twice is clearly wrong

```C
#if CONFIG_METEORLAKE

#elif CONFIG_LUNARLAKE

#elif CONFIG_LUNARLAKE

#endif
```

Not sure what `mod_cfg` does but it was clearly not tested.

But... wait: it looks like a 100% copy/paste? If it's not, then add `REM #` comment listing the differences which are nearly impossible to spot. If it is identical then don't copy/paste, use `#elif defined(CONFIG_LUNARLAKE) || ...`

Reply:
It is resolved already.